docs/apache-airflow/executor/kubernetes.rst

```diff
@@ -21,22 +21,22 @@
 Kubernetes Executor
 ===================
 
-The kubernetes executor is introduced in Apache Airflow 1.10.0. The Kubernetes executor will create a new pod for every task instance.
+The Kubernetes executor runs each task instance in its own pod on a Kubernetes cluster.
 
-Example kubernetes files are available at ``scripts/in_container/kubernetes/app/{secrets,volumes,postgres}.yaml`` in the source distribution (please note that these examples are not ideal for production environments).
-The volumes are optional and depend on your configuration. There are two volumes available:
+Use of persistent volumes is optional and depends on your configuration. There are two types of volumes you may configure:
 
 - **Dags**:
 
-  - By storing dags onto persistent disk, it will be made available to all workers
+  - By storing dags on a persistent volume, it will be made available to all workers
```

Comment:
it's a little unclear to me what this line is actually meant to help the user understand. in order to use a PV, wouldn't you also need to have set up git sync, so that you actually get dags onto the volume? but here it's sort of represented as an alternative to git sync?

Reply:
Totally possible to have a PV that is populated by CICD. That said, this whole section until `pod_template_file` feels a little out of place to me for KubenetesExecutor docs 🤷‍♂️. This likely made more sense when those example yaml files did exist.

Reply:
> Totally possible to have a PV that is populated by CICD

Good point.

> That said, this whole section until pod_template_file feels a little out of place to me for KubenetesExecutor docs 🤷‍♂️. 

I agree -- this kind of subject seems to make more sense in helm chart.  I can do another pass through with this in mind, and try to focus it just on the executor and if we need to point to helm chart docs where it gets more into implementation -- LMKYT

Reply:
yeah DAG deployment strategies should be out of this doc and we already have it for the Helm Chart

Reply:
OK so, after some reflection, one thought on this section.

Users don't need to use our helm chart, nor do they even need to be running airflow on k8s, in order to use k8s executor.  They just need access to a cluster.  E.g. you could have scheduler and webserver running on ECS, and send your pods to an eks cluster you have access to.  

So with that in mind, looking only at k8s executor, it doesn't seem unreasonable to keep this brief section on how to manage dags and logs.

Reply:
In that case, can we have a separate section before "Comparison with CeleryExecutor"?

Reply:
sure -- just reorganized, ptal